This renames the executables, so that the development packages can be installed alongside the release packages.
Index: carta-backend-beta-2021.3.5/CMakeLists.txt
===================================================================
--- carta-backend-beta-2021.3.5.orig/CMakeLists.txt
+++ carta-backend-beta-2021.3.5/CMakeLists.txt
@@ -193,11 +193,14 @@ if (CartaDefaultFrontendFolder)
     add_compile_definitions(CARTA_DEFAULT_FRONTEND_FOLDER="${CartaDefaultFrontendFolder}")
 endif (CartaDefaultFrontendFolder)
 
+set_target_properties(carta_backend PROPERTIES SUFFIX "_beta")
+
 install(TARGETS carta_backend
     RUNTIME DESTINATION bin)
 
 install(PROGRAMS "${CMAKE_SOURCE_DIR}/scripts/carta"
-    DESTINATION bin)
+    DESTINATION bin
+    RENAME carta_beta)
 
 # Tests
 option(test "Build tests." OFF)
Index: carta-backend-beta-2021.3.5/scripts/carta
===================================================================
--- carta-backend-beta-2021.3.5.orig/scripts/carta
+++ carta-backend-beta-2021.3.5/scripts/carta
@@ -10,4 +10,4 @@ fi
 
 export SERVER_IP
 
-carta_backend $@
+carta_backend_beta $@
Index: carta-backend-beta-2021.3.5/static/carta.desktop
===================================================================
--- carta-backend-beta-2021.3.5.orig/static/carta.desktop
+++ carta-backend-beta-2021.3.5/static/carta.desktop
@@ -1,8 +1,8 @@
 [Desktop Entry]
 Type=Application
-Name=CARTA
+Name=CARTA (beta)
 GenericName=Cube Analysis and Rendering Tool for Astronomy
-Exec=carta
+Exec=carta_beta
 Terminal=true
 Icon=cartaviewer
 Categories=Science
